src/main/java/com/fasterxml/jackson/databind/cfg/MapperConfig.java

```diff
@@ -367,6 +367,18 @@ public BeanDescription introspectDirectClassAnnotations(Class<?> cls) {
 
     /**
      * Accessor for default property inclusion to use for serialization,
+     * considering possible per-type override for given base type and
+	 * possible per-type override for given property type.<br>
+     * NOTE: if no override found, defaults to value returned by
+     * {@link #getDefaultPropertyInclusion()}.
+     *
+     * @since 2.8.8
+     */
+    public abstract JsonInclude.Value getDefaultPropertyInclusion(Class<?> baseType,
```

Comment:
Agreed, that it shouldn't be merged as-is. As mentioned on the initial PR comment, this is just intended to be a proof-of-concept.

As far I understood the usage of these methods, the `baseType` refers to the type of bean containing the property and the additional type argument refers to the type of the property itself. The existing methods cannot look up the override configuration for a single property as they are not aware of its type.